docs/nlu/using-bert.rst

```diff
@@ -0,0 +1,157 @@
+:desc: Find out how to use only Bert inside of Rasa NLU.
```

Comment:
this goes into the meta tag and is important for SEO. Can be a few sentences, and should also include other important keywords like GPT-2. @DominikRos has experience writing these

Reply:
I've added Huggingface now too. Would love to hear about more buzzwords to add.